assets/src/editor/events/dates-and-times/editor-date/filter-bar/with-date-entities-list-filter-state.js

```diff
@@ -23,8 +23,10 @@ export default createHigherOrderComponent(
 				datesSortedBy = 'chronologically',
 				displayDates = 'start',
 				searchDateName = '',
-				datesPerPage = 6,
-				datesView = 'grid',
+				datesPerPage,
+				datesView,
+				defaultDatesListView,
+				defaultDatesListPerPage,
```

Comment:
Why have two sets of props here for per page and view type?  Establishing this pattern becomes hard to maintain if in the future other props having defaults are also implemented (meaning everytime such prop is added, another default prop is included).  I think it might be better to lift up the state of default management and only pass through what the set prop is OR internally manage the default in this component.